notebooks/ingestion/Loading_CSVParsers.ipynb

```diff
@@ -4,49 +4,69 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Loading the data into the DB, using CSV Parsers (2/3).\n",
+    "# Ingest Cookbook (CSVParser).\n",
     "\n",
-    "Following the concepts on the different [means](/HowToGuides/start/Ingestion) of ingesting the data, we will build an example using the CSV Parser method in this notebook.\n",
+    "Following the concepts on the different [means](https://docs.aperturedata.io/HowToGuides/Ingestion/Ingestion#ingestion-using-data-from-csv-files) of ingesting the data, we will build an example using the CSV Parser method in this notebook.\n",
     "\n",
     "We will use the [Cookbook dataset](https://docs.google.com/spreadsheets/d/1G1HPG3Dxx5W39OD6b74wMHvWupD7N-DLUbV7tD5owx8/edit?usp=sharing) to have the data persisted onto ApertureDB instance.\n",
     "\n",
-    "We convert the first 3 sheets on the source into 3 CSV files which are compatible with the CSV Parsers in the SDK. For a comprehensive understanding of the functions of the various CSV Parsers, refer to this [page](/category/csv-parsers).\n"
+    "The script converts the first 3 sheets on the source into 3 CSV files which are understood by the CSV Parsers in the SDK. For a more in-depth understanding of the various CSV Parsers, refer to this [page](https://docs.aperturedata.io/category/csv-parsers).\n",
+    "\n",
+    ":::note\n",
+    "\n",
+    "The guide assumes that the SDK has been setup as per the instructions.\n",
+    "\n",
+    "Colab one time setup [instructions](/Setup/client/notebooks?notebooks=colab).\n",
+    "\n",
+    "Non ephemeral jupyter server one time setup [instructions](/Setup/client/notebooks?notebooks=notebook).\n",
+    "\n",
+    ":::\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Data Models\n",
-    "\n",
-    "A popular way to define the schema in python is using pydantic, and we shall use the same to create the associations of our Cookbook.\n"
+    "## Some needed resources."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
```

Comment:
do we want outputs cleared?

Reply:
That's a good question! I have always cleared my outputs because I felt like it can sometimes make it harder for users to read if they want to see all of the steps before going through

Reply:
cleaned up.